kube-derive/src/custom_resource.rs

```diff
@@ -420,6 +434,7 @@ pub(crate) fn derive(input: proc_macro2::TokenStream) -> proc_macro2::TokenStrea
                         "openAPIV3Schema": schema,
                     },
                     "additionalPrinterColumns": columns,
+                    #selectable
```

Comment:
You can do the same as `additionalPrinterColumns` here and do:
```suggestion
                    "additionalPrinterColumns": columns,
                    "selectableFields": selectable,
```
to avoid the aggregation if above. the [schema for versions](https://docs.rs/k8s-openapi/latest/k8s_openapi/apiextensions_apiserver/pkg/apis/apiextensions/v1/struct.CustomResourceDefinitionVersion.html) will not include the field in the output because it's wrapped in a [non-serialized option](https://docs.rs/k8s-openapi/latest/src/k8s_openapi/v1_31/apiextensions_apiserver/pkg/apis/apiextensions/v1/custom_resource_definition_version.rs.html#162-197)

Reply:
I think I hit an error in 1.26 test run when trying this approach in previous commits. https://github.com/kube-rs/kube/actions/runs/11348769321/job/31563326633

Reply:
Ah, fair.